app/commands/github/issue_label/create_or_update.rb

```diff
@@ -6,7 +6,7 @@ class CreateOrUpdate
       initialize_with :issue, :label
 
       def call
-        ::Github::IssueLabel.create_or_find_by!(issue: issue, label: label)
+        ::Github::IssueLabel.create_or_find_by!(issue: issue, name: label)
```

Comment:
I've decided to rename the column to match the github property name and it is a little clearer. Label.label is weird